### PR TITLE
Add a script for creating edx-platform conversion INCR tickets.

### PR DESCRIPTION
**Pull request by feanil**

### CODE REVIEW

**Comment by feanil** [comment]
@estute not sure if it makes sense to have this here, what do you think?

File: scripts/create_file_batches.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    to find files that still need to be converted.
+    """
+
+    for filename in filter_python_files(files):
+        ## Check to see if the file has already been converted.
+        #import pudb; pu.db
+        file_path = os.path.join(root, filename)
+        file_is_python_3 = False
+        with open(file_path, 'r') as f:
+            for line in f.readlines():
```
Comment (nedbat) [inline scripts/create_file_batches.py]: `for line in f:`

**Comment by estute** [comment]
Sure, this repo tends to be a catch-all for Tools/Testeng scripts. Not sure if you saw this https://github.com/edx/testeng-ci/blob/master/scripts/create_incr_tickets.py. It doesn't use the JIRA template or check for python-3, so perhaps we should remove it?

**Comment by feanil** [comment]
I did see that and tried to use it initially but eventually opted for just simpler chunks so that I could get all the work ticketed all at once.  I can remove the other script if it doesn't make sense to keep.  Honestly, I felt the same about this script.  It's not too hard to write and is fairly specific so I'm not sure if I should just plan on re-writing it if we need it again. I decided to push it up just in case people ask how I created all the PRs that I just added to INCR.  Let me know if you think we should do something different.

**Pull request closed without merging**